libstorage/drivers/storage/rbd/utils/utils_test.go

```diff
@@ -75,3 +75,43 @@ func TestParseMonitorAddresses(t *testing.T) {
 		assert.True(t, test.ip.Equal(ip[0]))
 	}
 }
+
+func TestArgsClient(t *testing.T) {
+	tests := []struct {
+		args    string
+		present bool
+	}{
+		{
+			args:    "--id myuser",
+			present: true,
+		},
+		{
+			args:    "--user myuser",
+			present: true,
+		},
+		{
+			args:    "--name client.myuser",
+			present: true,
+		},
+		{
+			args:    "--cluster cluster2",
+			present: true,
```

Comment:
This should be `present: false`. There is only a `cluster` flag in here, and therefore no client is detected. This should actually fail if you run the tests, e.g. :
`go test -v github.com/thecodeteam/rexray/libstorage/drivers/storage/rbd/utils`

Do you not see the tests fail?

I also had a 6th variation in the sample I pasted:
```
		{
			args:    "--cluster cluster2 --id ceph",
			present: true,
		},
```

That was just to make sure we catch a mixed example.